src/oaklib/converters/obo_graph_to_fhir_converter.py

```diff
@@ -46,11 +46,45 @@
 
 @dataclass
 class OboGraphToFHIRConverter(DataModelConverter):
-    """Converts from OboGraph to FHIR."""
+    """Converts from OboGraph to FHIR.
+
+    - An ontology is mapped to a FHIR `CodeSystem <https://build.fhir.org/codesystem.html>`_.
+    - Each node in the OboGraph is converted to a FHIR Concept.
+    - Each CURIE/URI in the OboGraph is treated as a CURIE when it becomes a code (e.g. "HP:0000001")
```

Comment:
@cmungall I imagine we will be in discussion on this for a little bit. In most situations, the consensus that I got from the FHIR docs, FHIR RDF group, and others, is that we should not use CURIEs and use something like [FHIR Coding](https://build.fhir.org/datatypes.html#Coding)s.